lib/file.js

```diff
@@ -825,10 +826,11 @@ const writeout = async (root, sources) =>
     Promise.all(
         sources.map(async source => {
             const dir = path.join(root, source.path.asDirectory({local: false, posix: false}))
+            const output_file = cds.env.typer.output_d_ts_files ? 'index.d.ts' : 'index.ts'
```

Comment:
Please use:
```suggestion
            const output_file = configuration.outputDTsFiles ? 'index.d.ts' : 'index.ts'
```
this mixes CLI and cds.env and allows passing the parameter as both snake_case and camelCase.

Please also use camelCase for variables (you consistently used `output_file`, it should be `outputFile`). That one is partially on me, I should have this in our eslint config.

Reply:
I have added the camelcase rule to the eslint config, if you merge in `main`, it should hopefully help finding the snake_case variables/ parameters.

Reply:
should _camelCase_ be used for the object _properties_ too? e.g. here -> [test.config.js#R11-R12](https://github.com/cap-js/cds-typer/pull/567/files#diff-324e5ae6ed1d64846d943bc35a8a7394f2099af356ca5d9e30413648a3ae124fR11-R12)

secondly, if you missed it could you please leave any comment / opinion on [this change](https://github.com/cap-js/cds-typer/pull/567#discussion_r2355548287) for the smoke test?.. 🙂

Reply:
right, but wanted to do smth _more compact, less noisy_ 😅

Reply:
so you'd use the `options` parameter and access the properties thereof instead of destructuring it? I don't see a problem with that, and I don't think it's a hack. Or am I missing something?

Reply:
I am unclear on why you use the snake_case style in your `const CDS_TEST_CONFIG_OPTIONS`to begin with. I have not seen any place where just using 

```js
const CDS_TEST_CONFIG_OPTIONS = [
    { outputDTsFiles: false, outputFile: 'index.ts' },
    { outputDTsFiles: true, outputFile: 'index.d.ts' },
]
```
would be a problem 🤔